optimum/gptq/quantizer.py

```diff
@@ -340,10 +343,10 @@ def quantize_model(self, model: nn.Module, tokenizer: Any):
 
         def store_input_hook(_, input, *args):
             kwargs = args[0]
-            input = input[0]
+            input = (input[0],) if len(input) < 3 else input
```

Comment:
Let's not have the condition and just use the whole input. 
```suggestion

```

Reply:
Agree, thanks